src/blocks/mrc_set_python_variable.ts

```diff
@@ -23,21 +23,27 @@
 import * as Blockly from 'blockly';
 import { Order } from 'blockly/python';
 
-import * as pythonUtils from './utils/generated/python';
-import { createFieldDropdown } from '../fields/FieldDropdown';
-import { createFieldNonEditableText } from '../fields/FieldNonEditableText';
 import { getAllowedTypesForSetCheck } from './utils/python';
+import * as variable from './utils/variable';
 import { ExtendedPythonGenerator } from '../editor/extended_python_generator';
+import { createFieldDropdown } from '../fields/FieldDropdown';
+import { createFieldNonEditableText } from '../fields/FieldNonEditableText';
 import { MRC_STYLE_VARIABLES } from '../themes/styles';
+import * as toolboxItems from '../toolbox/items';
+
 
 // A block to set a python variable.
 
 export const BLOCK_NAME = 'mrc_set_python_variable';
 
+enum VariableKind {
```

Comment:
Why is VariableKind defined in both mrc_get_python_variable.ts and here?   It seems like since they are the same they should be shared.

Reply:
I'll put in a TODO so we don't forget.